src/Our.Umbraco.Ditto/Extensions/TypeInferenceExtensions.cs

```diff
@@ -8,7 +10,7 @@
     /// Extensions methods for <see cref="T:System.Type"/> for inferring type properties.
     /// Most of this code was adapted from the Entity Framework
     /// </summary>
-    internal static class TypeInferenceExtensions
+    public static class TypeInferenceExtensions
```

Comment:
Why mark as `public`? (we'd prefer not to - makes it difficult to manage the inner-workings in future)

If it is for the unit-tests project, then we can use the `InternalsVisibleTo` attribute:

    [assembly: InternalsVisibleTo("Our.Umbraco.Ditto.Tests")]